File: web-ui/src/app/model/operator.ts
export interface Operator {
  id: number;
  name: string;
  numOfInstances: number;
  throughput: number; // tuples/s
  latency: number;  // ms
  explorationStrategy: string;
  schedulingGranularity: string;
  abortHandling: string;
}

File: web-dashboard/src/main/java/object/DAO/Operator.java
package object.DAO;

import lombok.Data;
import lombok.ToString;

@Data
@ToString(callSuper = true)
public class Operator {
    private String id;
    private String name;
    private String numOfInstances;
    private Float throughput;
    private Float latency;
    private String explorationStrategy;
    private String schedulingGranularity;
    private String abortHandling;
}
